### ISSUE TITLE
incorrect cred reference in how-to doc

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6211]
|<img alt="ironfroggy's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| ironfroggy reported|
|-|-|
|Trac ID|trac#6211|
|Type|defect|
|Created|2012-12-17 02:37:03Z|
|Branch|https://github.com/twisted/twisted/tree/cred-howto-example-misimports-6211|

http://twistedmatrix.com/documents/11.0.0/core/howto/cred.html

In the first code example here there is an import as

**from twisted.cred import credentials, error**

which is later used as

**cred.credentials.UsernamePassword(user, password),**

Which appears to be a mistake. it should simply refer to "credentials"

<details><summary>Searchable metadata</summary>

```
trac-id__6211 6211
type__defect defect
reporter__ironfroggy ironfroggy
priority__normal normal
milestone__ 
branch__branches_cred_howto_example_misimports_6211 branches/cred-howto-example-misimports-6211
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1355711823000000 1355711823000000
changetime__1356406771000000 1356406771000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [36576]) Branching to 'cred-howto-example-misimports-6211'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

I moved the code into a separate file where tools like pyflakes can be applied to it.  Then I fixed the reported issue and a couple other very minor coding standard issues.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [36640]) Merge cred-howto-example-misimports-6211

Author: exarkun
Reviewer: thijs
Fixes: [#6211](https://github.com/twisted/twisted/issues/6211)

Split a cred example out into a dedicated listing file where the code is more easily
tested (and static analysis tools are more easily applied to it) and fix an incorrect
name in the code which led to it not operating properly.
